packages/currency/src/token.ts

```diff
@@ -0,0 +1,56 @@
+import { RequestLogicTypes } from '@requestnetwork/types';
+import { Currency } from './currency';
+import { getSupportedERC20Tokens } from './erc20';
+
+/**
+ * @class Token extends Currency to reference the symbol
+ */
+export class Token extends Currency {
+  constructor(
+    public value: string,
+    public type: RequestLogicTypes.CURRENCY,
+    public symbol: string,
+    public network?: string,
```

Comment:
the parent class already has those fields 

```suggestion
    value: string,
    type: RequestLogicTypes.CURRENCY,
    public readonly symbol: string,
    network?: string,
```